docs/dev/run_the_etl.rst

```diff
@@ -0,0 +1,88 @@
+.. _run-the-etl:
+
+=======================
+Walk, Don't Run the ETL
```

Comment:
Cute but confusing.

Reply:
I honestly found it confusing! Not for a long time, but it made the document less [skimmable](https://www.writethedocs.org/guide/writing/docs-principles/#skimmable), less immediately understandable, even to me who knows all this stuff at some level. I think it will be frustrating / annoying for someone who is just trying to figure out how to use the software.